File: backend/services/stockservice/src/main/java/com/api/stockservice/domain/event/CategoryEvents/UpdateCategoryEvent.java
package com.api.stockservice.domain.event.CategoryEvents;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UpdateCategoryEvent {
    private String CategoryId;
    private String Name;
    private String Thumbnail;
}

File: backend/services/MarketplaceService/src/MarketplaceService.Domain/Events/CategoryEvent/UpdateCategoryEvent.cs
using EventsContracts.EventsContracts;

namespace MarketplaceService.Domain.Events.CategoryEvent
{
    internal class UpdateCategoryEvent : IUpdateCategoryEvent
    {
        public string CategoryId { get; set; }

        public string Name { get; set; }

        public string Thumbnail { get; set; }
    }
}
